static/js/settings_account.js

```diff
@@ -184,6 +184,63 @@ exports.set_up = function () {
         });
     });
 
+    function show_reset_email_loader() {
+        var spinner_elem = $(".loading_indicator_spinner");
+        spinner_elem.html(templates.render("loader"));
+        spinner_elem.show();
+        $('.loading_indicator_message').show();
+    }
+
+    function hide_reset_email_loader() {
+        var spinner_elem = $(".loading_indicator_spinner");
+        spinner_elem.html(null);
+        spinner_elem.hide();
+        $('.loading_indicator_message').hide();
+    }
+
+    function send_password_reset() {
+        var email = people.my_current_email();
+        var form_data = new FormData();
+
+        form_data.append("email", email);
+        channel.post({
+            url: '/accounts/password/reset/',
+            data: form_data,
+            cache: false,
+            processData: false,
+            contentType: false,
```

Comment:
Do we need these last 3 fields?  We don't usually use them...

Reply:
I replace `data` from `FormData()` to dict, Since we have only one `email` variable in `data`. As well as, removed three variables which are not required in this case.